### PR TITLE
chore(uec): implement bounded hardening closure

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Implements the bounded UEC proposal packet end to end across canonical governance, disclosure, runtime evidence, and validation surfaces. This adds the bounded-hardening successor release bundle, fresh review/audit lineage, packet-specific governance contracts, and the validator/report family needed to certify the bounded admitted-universe claim.

## Why

The repo already supported a bounded live claim, but the proposal packet and incorporated audit identified remaining normalization and hardening debt: stale review lineage, missing audit crosswalk, distributed non-authority rules, stale contract-version declarations, bounded evaluator/depth reporting, and projection-shell ambiguity. No-Issue: bounded UEC packet implementation requested directly in-session.

## How

Followed the updated bounded UEC packet as an execution spec rather than a design memo. Landed the new governance/control surfaces (`current-audit-crosswalk`, `non-authority-register`, ingress manifest, dossier/evaluator/hidden-check contracts), refreshed active bounded wording and release-lineage pointers, minted the fresh bounded-hardening review packet and retained audit bundle, generated the successor bounded closure release bundle, added packet validators, and recertified from both the working tree and a freshly materialized pass-2 workspace.

## Profile Selection Receipt

- Semantic version source(s): repo ingress + workspace charter (`pre-1.0`)
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): no downtime; one bounded active release line; fresh review/audit/release bundles generated under new roots; rollback remains pointer-based through release-lineage and effective projections
- Tie-break status: none required
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan

- Workstreams and scope: claim wording normalization; audit/review lineage refresh; non-authority and retirement normalization; ingress/projection-shell controls; dossier/evaluator/hidden-check hardening; successor bounded release regeneration; certification
- Public interfaces/types/contracts affected: support-target claim mode, release-lineage claim scope, claim-truth conditions, retirement/non-authority governance, ingress manifest, support dossier sufficiency contracts, packet-specific validator scripts, architecture-conformance workflow
- Test scenarios: local packet validator sweep; fresh-materialized workspace pass-2 validator sweep; support-target live claim parity; bounded wording; audit crosswalk; review freshness; dossier depth; evaluator diversity; hidden-check breadth; contract-version coherence; projection-shell boundary checks
- Assumptions/defaults: admitted live support universe remains frozen at six tuples; no support widening occurs in this PR; packet-specific pass-2 validation can use a freshly materialized reduced workspace containing all files the validator suite reads

## Impact Map (code, tests, docs, contracts)

- Code/workflows: `.github/workflows/architecture-conformance.yml`
- Validators/scripts: `/.octon/framework/assurance/runtime/_ops/scripts/validate-*bounded*`, `validate-audit-crosswalk.sh`, `validate-review-packet-freshness.sh`, `validate-non-authority-register.sh`, `validate-retirement-register-depth.sh`, `validate-ingress-manifest-parity.sh`, `validate-support-dossier-evidence-depth.sh`, `validate-evaluator-diversity.sh`, `validate-hidden-check-breadth.sh`, `validate-contract-family-version-coherence.sh`, `validate-projection-shell-boundaries.sh`
- Contracts/docs: claim-truth conditions, family READMEs, registry, closeout reviews, retirement register, support dossiers/admissions, ingress docs/manifest, non-authority and projection-shell policies, prompt library, proposal packet
- Evidence/disclosure: fresh build-to-delete packet, fresh bounded audit bundle, successor release bundle `2026-04-09-uec-bounded-hardening-closure`, new repo-shell boundary-sensitive retained run bundle

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

This PR is intentionally large because it carries the full packet, packet-contained audit, canonical implementation surfaces, and successor release/evidence roots in one branch-level migration. The one narrow compromise is that the repo-shell boundary-sensitive representative run was minted by adapting the existing retained bundle shape rather than re-executing an external live scenario from scratch.

## Testing

Automated/local:
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-bounded-claim-nomenclature.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-audit-crosswalk.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-review-packet-freshness.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-non-authority-register.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-retirement-register-depth.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-ingress-manifest-parity.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-dossier-evidence-depth.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-evaluator-diversity.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-hidden-check-breadth.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-contract-family-version-coherence.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-projection-shell-boundaries.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-live-claims.sh`

Fresh-workspace pass 2:
- repeated the same validator suite from `/tmp/octon-bounded-pass2`, a freshly materialized workspace containing the validator inputs and helper script roots

## Rollout

Atomic pointer-based promotion only. The bounded successor release becomes active through `release-lineage.yml` and effective closure projections; prior active releases remain historical evidence only.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [x] Medium [ ] High
- Rollback plan: repoint `release-lineage.yml` and effective closure projections to the prior active release; leave the successor bundle on disk as unpromoted evidence
- Rollback handle: `2026-04-09-uec-hardening-recertification`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: none outside repo-local YAML/schema/governance contract surfaces
- Threat model update/link: packet adds projection-shell boundary policy, non-authority register, and bounded wording/audit freshness checks; no separate threat-model doc added

## Observability and Performance

- Traces/logs/metrics for changed flows: validator outputs and release-close reports under the successor bounded closure bundle
- Representative traces for high risk changes: bounded hardening validation reports and the new repo-shell boundary-sensitive retained run bundle
- Performance or bundle impact: negligible runtime impact; repository evidence footprint increases materially because the packet, fresh review packet, fresh audit bundle, and successor release bundle are retained together

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: packet prompt and proposal artifacts authored in-repo; no third-party generated code introduced

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] All review conversations resolved

## Screenshots / Notes

No UI screenshots. The key reviewer surfaces are the successor release bundle, fresh review packet, fresh retained audit bundle, and the packet-specific validator scripts.

### CODE REVIEW

File: .octon/instance/ingress/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - .octon/framework/constitution/charter.yml
+  - .octon/framework/constitution/obligations/fail-closed.yml
+  - .octon/framework/constitution/obligations/evidence.yml
+  - .octon/framework/constitution/precedence/normative.yml
+  - .octon/framework/constitution/precedence/epistemic.yml
+  - .octon/framework/constitution/ownership/roles.yml
+  - .octon/framework/constitution/contracts/registry.yml
+  - .octon/instance/charter/workspace.md
+  - .octon/instance/charter/workspace.yml
+  - .octon/framework/agency/runtime/agents/orchestrator/AGENT.md
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/instance/ingress/manifest.yml:14]: Fixed. `.octon/instance/ingress/AGENTS.md` now includes `.octon/framework/agency/runtime/agents/orchestrator/AGENT.md` in the mandatory read order so it stays in parity with the ingress manifest.

File: .github/workflows/architecture-conformance.yml [hunk 2]
```diff
@@ -200,6 +200,29 @@ jobs:
...
+
+      - name: Install yq
+        run: sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.50.1/yq_linux_amd64 && sudo chmod +x /usr/local/bin/yq
+
+      - name: Validate bounded hardening packet surfaces
+        run: |
+          bash .octon/framework/assurance/runtime/_ops/scripts/validate-bounded-claim-nomenclature.sh
+          bash .octon/framework/assurance/runtime/_ops/scripts/validate-audit-crosswalk.sh
+          bash .octon/framework/assurance/runtime/_ops/scripts/validate-review-packet-freshness.sh
+          bash .octon/framework/assurance/runtime/_ops/scripts/validate-non-authority-register.sh
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .github/workflows/architecture-conformance.yml:231]: Fixed. The architecture workflow path filters now cover the bounded hardening inputs, including ingress surfaces plus the non-authority, retirement, admission, dossier, and disclosure paths read by the new job.

File: .octon/framework/assurance/runtime/_ops/scripts/validate-ingress-manifest-parity.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+errors=0
+fail() { echo "[ERROR] $1"; errors=$((errors + 1)); }
+pass() { echo "[OK] $1"; }
+
+echo "== Ingress Manifest Parity Validation =="
+[[ -f "$MANIFEST" ]] && pass "ingress manifest exists" || fail "missing ingress manifest"
+
+for target in "$OCTON_ADAPTER" "$ROOT_ADAPTER" "$CLAUDE_ADAPTER"; do
+  if cmp -s "$OCTON_ADAPTER" "$target"; then
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/validate-ingress-manifest-parity.sh:36]: Fixed. `validate-ingress-manifest-parity.sh` now reads the manifest contents and checks the mandatory orchestrator read plus the branch closeout prompt instead of only validating file presence/parity.
